player-small.js

```diff
@@ -302,6 +302,18 @@ var CPlayer = function() {
         return mCurrentCol / mSong.numChannels;
     };
 
+    // Create a AudioBuffer from the generated audio data
+    this.createAudioBuffer = function(context) {
+        var buffer = context.createBuffer(2, mMixBuf.length / 2, 44100);
```

Comment:
Minification optimization: Use `mNumWords` instead of `mMixBuf.length`.

Reply:
Done!